packages/angular/cli/models/schematic-command.ts

```diff
@@ -7,17 +7,36 @@
  */
 
 // tslint:disable:no-global-tslint-disable no-any
-import { JsonObject, experimental } from '@angular-devkit/core';
-import { normalize, strings, tags, terminal, virtualFs } from '@angular-devkit/core';
+import { JsonObject,
```

Comment:
Please reformat your file not using clang format. I think @filipesilva uses prettier which works with the correct settings.

Reply:
Actually, I formatted it manually... @filipesilva @hansl  Any commands can I use? I tried 'npm run admin -- lint --fix',  it doesn't fix the 'import'.

Reply:
I don't use `prettier` but rather VS Code auto format which is not enforceable since it's editor specific.

In general for multiline imports everywhere else in the CLI we do it as below, so that's my preference.
```
import {
  import1,
  import2,
  // etc
} from 'package';
```

Reply:
Update import accordingly.